pkg/providers/aws/provider_postgres.go

```diff
@@ -211,8 +211,46 @@ func (p *PostgresProvider) CreatePostgres(ctx context.Context, pg *v1alpha1.Post
 		logger.Infof("created security group %s", aws.StringValue(securityGroup.StandaloneSecurityGroup.GroupName))
 	}
 
+	session := rds.New(sess)
 	// create the aws RDS instance
-	return p.createRDSInstance(ctx, pg, rds.New(sess), ec2.New(sess), rdsCfg, isEnabled)
+	postgres, status, err := p.createRDSInstance(ctx, pg, session, ec2.New(sess), rdsCfg, isEnabled)
+	if err != nil {
+		errMsg := "failed to create rds instance"
+		return nil, status, errorUtil.Wrap(err, errMsg)
+	}
+
+	// Retrieve service maintenance updates, create and export Prometheus metrics
+	output, err := session.DescribePendingMaintenanceActions(&rds.DescribePendingMaintenanceActionsInput{ResourceIdentifier: rdsCfg.DBInstanceIdentifier})
+	if err != nil {
+		errMsg := fmt.Sprintf("failed to get pending maintenance information for RDS with identifier : %s", rdsCfg.DBInstanceIdentifier)
+		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
+	}
+	for _, rpma := range output.PendingMaintenanceActions {
+		// todo if the resource identifier does not equal the arn of the instance
+		/*if !rpma.ResourceIdentifier
+			continue
+		}*/
+		for _, pmac := range rpma.PendingMaintenanceActionDetails {
```

Comment:
I haven't tested this loop yet or the ApplyPendingMaintenanceAction function yet.